src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,137 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.utility.RegistrationException;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static List<User> storage;
+    private static User initialUser;
+    private static User properToRegistUser;
+
+    @BeforeAll
+    static void initializeRegistrationService() {
+        registrationService = new RegistrationServiceImpl();
+        storage = new ArrayList<>();
+        properToRegistUser = new User();
+    }
+
+    @BeforeEach
+    void setUp() {
+        initialUser = new User();
+        properToRegistUser.setLogin("dummy@37");
+        properToRegistUser.setAge(18);
+        properToRegistUser.setPassword("password1234");
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.people.remove(properToRegistUser);
+    }
+
+    @Test
+    void register_userIsNull_notOk() {
+        initialUser = null;
+        assertThrows(RegistrationException.class, () -> registrationService.register(initialUser));
+    }
+
+    @Test
+    void register_loginIsNull_notOk() {
+        initialUser.setLogin(null);
+        assertThrows(RegistrationException.class, () -> registrationService.register(initialUser));
+    }
+
+    @Test
+    void register_loginLessThanSixSymbols_notOk() {
+        initialUser.setLogin("dummy");
+        assertThrows(RegistrationException.class, () -> registrationService.register(initialUser));
+    }
+
+    @Test
+    void register_loginHasSixOrMoreSymbols_ok() {
```

Comment:
one _ok test of a user which has all valid data should be enough. You basically have the same test twice, only named differently